src/main/java/uk/gov/digital/ho/hocs/aws/MigrationLocalStackConfiguration.java

```diff
@@ -0,0 +1,52 @@
+package uk.gov.digital.ho.hocs.aws;
+
+import com.amazonaws.auth.AWSCredentialsProvider;
+import com.amazonaws.auth.AWSStaticCredentialsProvider;
+import com.amazonaws.auth.BasicAWSCredentials;
+import com.amazonaws.client.builder.AwsClientBuilder;
+import com.amazonaws.services.s3.AmazonS3;
+import com.amazonaws.services.s3.AmazonS3ClientBuilder;
+import com.amazonaws.services.sqs.AmazonSQSAsync;
+import com.amazonaws.services.sqs.AmazonSQSAsyncClientBuilder;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.cloud.aws.messaging.config.annotation.EnableSqs;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Primary;
+import org.springframework.context.annotation.Profile;
+
+@EnableSqs
+@Configuration
+@Profile({"migration-local"})
+public class MigrationLocalStackConfiguration {
+
+    private final AWSCredentialsProvider awsCredentialsProvider;
+    private final AwsClientBuilder.EndpointConfiguration endpoint;
+
+    public MigrationLocalStackConfiguration(@Value("${localstack.base-url}") String baseUrl,
+                                            @Value("${localstack.config.region}") String region) {
+        this.awsCredentialsProvider = new AWSStaticCredentialsProvider(new BasicAWSCredentials("test", "test"));
```

Comment:
There is actually an [AnonymousAWSCredentials](https://docs.aws.amazon.com/AWSJavaSDK/latest/javadoc/com/amazonaws/auth/AnonymousAWSCredentials.html) that can be used here.

Reply:
Also don't need `this.` here.

Reply:
Fixed